CMakeLists.txt

```diff
@@ -75,6 +72,7 @@ set(exported_libraries "")
 
 ## Put libbarrett headers in the include path
 include_directories("${PROJECT_SOURCE_DIR}/include/")
+include_directories("${PROJECT_BINARY_DIR}/include")
```

Comment:
What is the significance of the trailing "/" versus no slash on the include directory?